### DISCUSSION TITLE
CLAP support?

### DISCUSSION THREAD

**Discussion by dromer** [DISTRHO/DPF#377] [Ideas]
I know I know, not something you necessarily want to work on or are looking forward to.

However it would still be good to have a discussion on gathering interest and potential contributors.
Assuming you are willing to look at pull requests and such that would contribute to this.

Potentially this wouldn't even be such a huge effort as it's being said that the implementation is straightforward. If this could get feature parity with LV2 I think it would put some extra attention to DPF to create a wide range of target plugins.

**Comment by falkTX** [comment]
I was against it at first because efforts were better spent on LV2, but it is too late for that now.
So now is more like "let's finish VST3 first, so it is not a big mess of half-finished implementations again".

Could depend on financial situation too. I heard companies behind clap were willing to sponsor opensource development and plugin ports, for which DPF fits quite nicely. So if I get in a specially tough situation, I might investigate that in more details.

For now, yeah, let's focus on VST3 even though it is a royal pain to deal with.
It is mostly just to get feature parity and have it as actual, complete, official feature of DPF. It has been a long time coming.

PS: I do not actually count on someone else doing the dpf-clap implementation. Those things just don't typically happen, I would be quite surprised if it was otherwise.

**Comment by dromer** [comment]
CLAP is here and it works!

(not yet parity with LV2, after that maybe poly-mod/midi2/etc??)
